python/paddle/trainer_config_helpers/tests/layers_test.py

```diff
@@ -16,4 +16,4 @@
 
 if __name__ == '__main__':
     parse_config_and_serialize(
-        'trainer_config_helpers/tests/layers_test_config.py', '')
+        'trainer_config_helpers/tests/configs/test_BatchNorm3D.py', '')
```

Comment:
Do not change this line. Add `test_BatchNorm3D.py` in the file `python/paddle/trainer_config_helpers/tests/configs/file_list.sh`. 

And please merge 3DConv/3DPooling first, then update this PR.

Reply:
Done